static/hydrus.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Version=1.0
+Name=Hydrus Client
+Comment=Danbooru-like image tagging and searching system for the desktop
+Exec=hydrus-client
+Icon=/usr/lib/hydrus/static/hydrus_non-transparent.png
```

Comment:
Is it possible to use relative paths here? I don't think every use will have hydrus in `/usr/lib/hydrus`

Reply:
Actually nevermind.
Mixed up a Unit file and a Desktop file...

Best we could do is symlink the hydrus icons to a system-wide recognized icon location (like `~/.icons/hydrus` for example) and just put `Icon=hydrus_non-transparent.png` which would 100% be possible in a kind of client setup script that hydrus-dev is working on